packages/wrangler/src/cfetch/internal.ts

```diff
@@ -43,18 +44,19 @@ export async function fetchInternal<ResponseType>(
 
   const queryString = queryParams ? `?${queryParams.toString()}` : "";
   const method = init.method ?? "GET";
-  const response = await fetch(
-    `${getCloudflareAPIBaseURL()}${resource}${queryString}`,
-    {
-      method,
-      ...init,
-      headers,
-      signal: abortSignal,
-    }
-  );
+  const URL = `${getCloudflareAPIBaseURL()}${resource}${queryString}`;
+  logger.debug(URL);
+  const response = await fetch(URL, {
+    method,
+    ...init,
+    headers,
+    signal: abortSignal,
+  });
   const jsonText = await response.text();
   try {
-    return parseJSON<ResponseType>(jsonText);
+    const json = parseJSON<ResponseType>(jsonText);
+    logger.debug(JSON.stringify(json, null, "\t"));
```

Comment:
I'm not sure we want to debug every response here (especially as a json object). Instead, at call points, we log important fields.